tests/Utils/Models/Task.php

```diff
@@ -37,6 +37,11 @@ protected static function boot(): void
         });
     }
 
+    public function activity(): MorphMany
```

Comment:
This model also lacks a PHPDoc annotation for this relation.

Reply:
Sorry, added.